projects/ids-enterprise-ng/src/lib/autocomplete/soho-autocomplete.component.ts

```diff
@@ -147,7 +149,8 @@ export class SohoAutoCompleteComponent extends BaseControlValueAccessor<string>
     // Bind to jQueryElement's events
     this.jQueryElement
       .on('selected', (...args) => this.selected.emit(args))
-      .on('change', (e: any, args: any[]) => this.onChange(args));
+      .on('change', (e: any, args: any[]) => this.onChange(args))
+      .on('beforeopen', (e: any, args: SohoAutoCompleteEvent) => this.beforeopen.emit(args));
```

Comment:
@bthharper  do we want to have folks convert to running ep code outside ngZone?

Reply:
@pwpatton  - I assume none of autocomplete has been updated to correctly switch zones yet, but it might be a good idea to do that here.  I would have expected the event handler to swap back into the Angular zone.

Reply:
Maybe also add some types to the change event.